packages/next/pages/_document.js

```diff
@@ -16,7 +16,7 @@ export default class Document extends Component {
   static getInitialProps ({ renderPage }) {
     const { html, head } = renderPage()
     const styles = flush()
-    return { html, head, styles }
+    return { html, head, styles, timestamp: Date.now() }
```

Comment:
Can we disable this in prod, probably not too heavy but still not useful to have `Date.now` running on every request.

Reply:
totally